### PR TITLE
Update to FR translation

### PR DESCRIPTION
**Pull request by Cryotech06**
Updated to most recent english file.

### CODE REVIEW

File: src/status_im/translations/fr.cljs [hunk 1]
```diff
@@ -60,15 +76,23 @@
...
-   :profile-no-status                     "Pas de statut"
+   :update-status                         "Actualisez votre statut..."
+   :add-a-status                          "Ajoutez un statut..."
+   :status-prompt                         "Créez un nouveau statut pour partager vos intérêts avec tout le monde. Vous pouvez aussi utiliser les #hashtags."
    :add-to-contacts                       "Ajouter aux contacts"
+   :in-contacts                           "Dans les contacts"
+   :remove-from-contacts                  "Supprimer des contacts"
+   :start-conversation                    "Débuter une conversation"
+   :send-transaction                      "Envoyer une transaction"
+   :share-qr                              "Partager QR"
```
Comment (tancou) [inline src/status_im/translations/fr.cljs]: "QR" to "QRCode" seems to be better.

File: src/status_im/translations/fr.cljs [hunk 2]
```diff
@@ -78,52 +102,90 @@
    :image-source-gallery                  "Sélectionner dans la galerie"
    :image-source-cancel                   "Annuler"
 
+   ;;sharing
+   :sharing-copy-to-clipboard             "Copier vers le presse-papiers"
```
Comment (tancou) [inline src/status_im/translations/fr.cljs]: "Copier vers le presse-papier"

File: src/status_im/translations/fr.cljs [hunk 3]
```diff
@@ -78,52 +102,90 @@
...
+   :sharing-share                         "Partagez..."
```
Comment (tancou) [inline src/status_im/translations/fr.cljs]: "Partager..."

File: src/status_im/translations/fr.cljs [hunk 4]
```diff
@@ -78,52 +102,90 @@
...
+   
    ;settings
    :settings                              "Paramètres"
 
    ;contacts
    :contacts                              "Contacts"
    :new-contact                           "Nouveau contact"
+   :delete-contact                        "Supprimer contact"
+   :delete-contact-confirmation           "Ce contact sera supprimé de vos contacts"
+   :remove-from-group                     "Supprimer ce groupe"
```
Comment (tancou) [inline src/status_im/translations/fr.cljs]: "Supprimer de ce groupe"

File: src/status_im/translations/fr.cljs [hunk 5]
```diff
@@ -78,52 +102,90 @@
...
+   :edit-contacts                         "Modifier contacts"
+   :search-contacts                       "Rechercher contacts"
    :show-all                              "TOUT AFFICHER"
```
Comment (tancou) [inline src/status_im/translations/fr.cljs]: "Tout afficher"

**Comment by flexsurfer** [comment]
hi @Cryotech06, thank you for your contribution, could you please look at @tancou comments and squash your commits, thank you

**Pull request merged**